frontend/src/assets/fonts/Source_Code_Pro/Source-Code-Pro.LICENSE

```diff
@@ -1,93 +1,93 @@
-Copyright 2010, 2012 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.

```

Comment:
This diff is just due to line ending changes (CRLF -> LF). Same with the rest of the font license files.